cannot find module 'fs-extra'

Since updating to 1.4.0 I am getting the following error when running grunt cacheBust:

```
Loading "cachebust.js" tasks...ERROR
>> Error: Cannot find module 'fs-extra'
```

Downgrading to 1.3.0 fixes the issue for me.

--- a/package.json
+++ b/package.json
@@ -20,7 +20,6 @@
     "test": "grunt test"
   },
   "devDependencies": {
-    "fs-extra": "^0.30.0",
     "grunt": "^1.0.1",
     "grunt-contrib-clean": "^1.0.0",
     "grunt-contrib-copy": "^1.0.0",
@@ -37,5 +36,8 @@
     "cache",
     "bust",
     "bust assets"
-  ]
+  ],
+  "dependencies": {
+    "fs-extra": "^0.30.0"
+  }
 }

